client/src/components/App.jsx

```diff
@@ -4,21 +4,40 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faPlayCircle, faToggleOff, faPauseCircle } from '@fortawesome/free-solid-svg-icons';
 import ArtistImage from './ArtistImage';
 import PlayButton from './PlayButton';
+import ajax from '../../lib/ajax';
 
 library.add(faPlayCircle);
 library.add(faPauseCircle);
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      
+      image: [],
```

Comment:
If you are not storing an array in images it might be better to change image's type to match what will be stored in image or null